docker/client.py

```diff
@@ -306,7 +306,7 @@ def attach_socket(self, container, params=None, ws=False):
 
     def build(self, path=None, tag=None, quiet=False, fileobj=None,
               nocache=False, rm=False, stream=False, timeout=None,
-              custom_context=False, encoding=None, pull=True,
+              custom_context=False, encoding=None, pull=False,
```

Comment:
Luckily since this was broken anyway, setting it to False won't break anything. :)